check_perccli

```diff
@@ -85,7 +85,8 @@ def main():
     else:
         log.debug('SSH connection established')
 
-    stdin, stdout, stderr = ssh_client.exec_command('%s /call show all j' % args.perccli_path)
+    # we cd to the dir with the executable because newer versions of perccli depend on libstorelib.so which is in the same directory as perccli executable
+	stdin, stdout, stderr = ssh_client.exec_command('cd `dirname %s`; %s /call show all j' % (args.perccli_path, args.perccli_path))
```

Comment:
I'm running the same version on my box and I don't need to cd in...
```
[root@esxi01:~] pwd
/
[root@esxi01:~] /opt/lsi/perccli/perccli -V

      PercCli SAS Customization Utility Ver 007.0529.0000.0000 Sep 18, 2018

    (c)Copyright 2018, AVAGO Technologies, All Rights Reserved.


Exit Code: 0x00
[root@esxi01:~]
```
Are you sure this is required?

BTW, your code includes TABs instead of 4 spaces which is upsetting Python 3.*:
```
  File "./check_perccli", line 89
    stdin, stdout, stderr = ssh_client.exec_command('cd `dirname %s`; %s /call show all j' % (args.perccli_path, args.perccli_path))
                                                                                                                                   ^
TabError: inconsistent use of tabs and spaces in indentation```

Reply:
Thanks for taking the time on these detailed comments. Will fix the tab issue.

Yeah, on my 6.5 and 6.7 boxes I have to CD or it will not see the controller.

```
[root@vmhost3:~] pwd
/
[root@vmhost3:~] /opt/lsi/perccli/perccli show
./libstorelib.so: cannot open shared object file: No such file or directory
CLI Version = 007.0529.0000.0000 Sep 18, 2018
Operating system = VMkernel 6.7.0
Status Code = 0
Status = Success
Description = None

Number of Controllers = 0
Host Name = vmhost3.localdomain
Operating System  = VMkernel 6.7.0
StoreLib IT Version = 07.0600.0200.0600
StoreLib IR3 Version = 16.03-0


[root@vmhost3:~] cd /opt/lsi/perccli/
[root@vmhost3:/opt/lsi/perccli] ls -l
total 7308
----------    1 root     root        259179 Sep 17  2018 libstorelib.so
-r-xr-xr-x    1 root     root       7219708 Sep 17  2018 perccli
[root@vmhost3:/opt/lsi/perccli] ./perccli show
CLI Version = 007.0529.0000.0000 Sep 18, 2018
Operating system = VMkernel 6.7.0
Status Code = 0
Status = Success
Description = None

Number of Controllers = 1
Host Name = vmhost3.localdomain
Operating System  = VMkernel 6.7.0
StoreLib IT Version = 07.0600.0200.0600
StoreLib IR3 Version = 16.03-0

System Overview :
===============

----------------------------------------------------------------------------
Ctl Model           Ports PDs DGs DNOpt VDs VNOpt BBU  sPR DS EHS ASOs Hlth
----------------------------------------------------------------------------
  0 PERCH330Adapter     8   3   1     0   1     0 Msng On  3  N      0 Opt
----------------------------------------------------------------------------

Ctl=Controller Index|DGs=Drive groups|VDs=Virtual drives|Fld=Failed
PDs=Physical drives|DNOpt=DG NotOptimal|VNOpt=VD NotOptimal|Opt=Optimal
Msng=Missing|Dgd=Degraded|NdAtn=Need Attention|Unkwn=Unknown
sPR=Scheduled Patrol Read|DS=DimmerSwitch|EHS=Emergency Hot Spare
Y=Yes|N=No|ASOs=Advanced Software Options|BBU=Battery backup unit
Hlth=Health|Safe=Safe-mode boot



[root@vmhost3:/opt/lsi/perccli]

```